README.md

```diff
@@ -286,6 +286,16 @@ w4 = u.wallet(w1.id)
 w4.load_seed(file_path)
 ```
 
+## External Addresses
+
+Addresses that you do not control the key for can still be interacted with via the Platform SDK.
```

Comment:
Addresses that do not belong CDP wallets can still be interacted with via the Platform SDK.
"Not control the key" isnt accurate. They could be controlling the key, just not loaded on platform sdk